PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -1057,6 +1066,50 @@ void AliEmcalJetTask::LoadTrackEfficiencyFunction(const std::string & path, cons
   delete file;
 }
 
+/**
+ * Set the pt-dependent tracking efficiency depending on periodfrom a YAML file
+ * @param path Path to the file containing the TF1
+ * @param name Name of the TF1
+ */
+
+void AliEmcalJetTask::SetTrackingEfficiencyFromYAML(std::string period , std::string path ) {
```

Comment:
You should consider renaming this to `SetArtificalTrackingEfficiencyFromYAML` or `SetTrackingInefficiencyFromYAML`. This is a an additional tracking efficiency variation since the tracking efficiency is "built in" (for the data), or applied explicitly in reconstruction (for MC). I think calling it just tracking efficiency is confusing. But if others disagree, you can leave it as is

Reply:
done, I also agree, your suggestion is better for clarity